java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/nodes/WindowAggPlan.scala

```diff
@@ -174,6 +184,22 @@ object WindowAggPlan {
       distributionDf
     }
 
+    val minBlockSize = if (!ctx.getConf.enableWindowSkewExpandedAllOpt) {
+      var approxMinCount = Long.MaxValue
+      val rows = distributionDf.select(distinctCountColName).collect()
```

Comment:
We can get min by SQL min function instead of collecting data to compute in driver